janitor/functions.py

```diff
@@ -3236,6 +3236,49 @@ def groupby_agg(
     new_col = df.groupby(by)[agg_column_name].transform(agg)
     df_new = df.assign(**{new_column_name: new_col})
     return df_new
+    """
```

Comment:
The contents of this docstring should go in the block right below the function. That way it will get recognized as proper/valid docstring.